internal/proxy/proxy.go

```diff
@@ -73,7 +73,7 @@ func NewProxy(
 		tenant:           tenant,
 		wit:              wit,
 		idler:            idler,
-		bufferCheckSleep: 30,
+		bufferCheckSleep: 30 * time.Second,
```

Comment:
shouldn't the type be also changed to time.Duration ?

Reply:
It is already time.Duration

Reply:
thanks, `time.Sleep(p.bufferCheckSleep * time.Second)` in the previous code confused me.